WalletWasabi.Gui/Converters/NetworkColorConverter.cs

```diff
@@ -0,0 +1,37 @@
+using Avalonia;
+using Avalonia.Data.Converters;
+using Avalonia.Media;
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using WalletWasabi.Exceptions;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class NetworkColorConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (value is Network network)
+			{
+				return network switch
+				{
+					_ when network == Network.TestNet => Brush.Parse("#AE6200"),
```

Comment:
Perhaps these could be added to Styles.xaml as resource for consistency

Reply:
How? The following doesn't work for me:

![image](https://user-images.githubusercontent.com/127973/88996152-e96dd880-d2c2-11ea-9164-382be04f9bfd.png)

```c#
public object Convert(object value, Type targetType, object parameter, CultureInfo culture) =>
	Application.Current.Resources[value switch
	{
		Network network when network == Network.TestNet => "TestNetStatusBarBackground",
		Network network when network == Network.RegTest => "RegTestStatusBarBackground",
		_ => "ApplicationAccentBrushLow"
	}] as IBrush;
```

Basically because the `Application.Current.Resources[]` doesn't have the keys.

Reply:
I have tried using `Classes="value"` in XAML but as it turns out it's not supported in Avalonia (https://github.com/AvaloniaUI/Avalonia/issues/2427). That's pretty common approach in HTML+CSS world.

Reply:
I made a behavior that appends a class string to the target control, lemme find it